### DISCUSSION TITLE
自定义tidb审核引擎

### DISCUSSION THREAD

**Discussion by jimsoft1000** [hhyo/Archery#2612] [General]
在1.8.5版本，自定义了tidb数据库的支持，但是需要修改很多前端的html页面，目前数据字典，在线查询，在线提交工单执行什么都没有问题，唯一遗憾的是，工单执行之后，回滚SQL这块不知道怎么处理。

![企业微信截图_1714959065603](https://github.com/hhyo/Archery/assets/74498996/70ea346c-96ee-4339-921d-d677b56468ef)

**Comment by sona-yuan** [comment]
tidb不允许备份即可
